ui/tests/integration/components/oidc/scope-form-test.js

```diff
@@ -176,14 +166,8 @@ module('Integration | Component | oidc/scope-form', function (hooks) {
     await click('[data-test-oidc-scope-example]');
     assert.dom(MODAL('title')).hasText('Scope template', 'Modal title renders');
     assert.dom(MODAL('text')).hasText('Example of a JSON template for scopes:', 'Modal text renders');
-    assert
-      .dom('#scope-template-modal [data-test-copy-button]')
-      .hasAttribute(
-        'data-test-copy-button',
-        exampleTemplate,
```

Comment:
Should we keep an assertion that the copy value is what we expect? There's been some issues in the past with code editors in modals working improperly

Reply:
Oh that's good to know, and it makes sense to add some tests, especially when there's been some weird behavior in the past. I will add some assertions to verify the content of the `CodeBlock` matches what we are expecting in a followup PR, since this one has already been merged. Thanks Claire!